.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image:  cimg/node:16.18
+      - image: cimg/node:18.18.1
```

Comment:
Latest Long-Term Support Version: [20.9.0 (includes npm 10.1.0)](https://nodejs.org/en/download) - no breaking changes from v18 (or v16) - [ref](https://www.itmagination.com/blog/introducing-node-js-20-new-features-updates-and-improvements) - maybe we can upgrade to that?